scripts/build-private.sh

```diff
@@ -32,4 +32,17 @@ cmake "-DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE" "-DSUNSHINE_EXECUTABLE_PATH=$SUNSHI
 
 make -j ${nproc}
 
-./gen-deb
+# Get preferred package format
+if [ "$1" == "-rpm" ]
+then
+  echo "Packaging in .rpm format."
+  ./gen-rpm
```

Comment:
Don't we need the `-d` argument? I thought that was already added?

Reply:
Yeah, we do.